frontend/src/pages/CompareV1.tsx

```diff
@@ -244,6 +244,18 @@ class CompareV1 extends Page<{}, CompareState> {
         `Failed loading ${failingRuns.length} runs, last failed with the error: ${lastError}`,
       );
       return;
+    } else if (
+      runs.length > 0 &&
```

Comment:
This should be an invariant anyway for this page, but if this page is somehow directly displayed with no runs, still prevents this banner from unnecessarily being set.

Reply:
Resolving this discussion as the PR has been reviewed.